startup/gui/renderPassEditor.py

```diff
@@ -113,3 +113,13 @@
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:GI_transmission_depth", "Ray Depth", "Transmission" )
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:GI_volume_depth", "Ray Depth", "Volume" )
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:auto_transparency_depth", "Ray Depth", "Transparency" )
+
+# Register the default grouping function used to display render passes in a hierarchy.
+# This groups render passes based on the first token in their name delimited by "_".
+def __defaultPathGroupingFunction( renderPassName ) :
+
+	return IECore.InternedStringVectorData(
+		[ renderPassName.split( "_" )[0] ] if "_" in renderPassName else []
```

Comment:
It's still bothering me that `:` is our natural namespacing separator everywhere else, but I don't see any alternatives right now.

Reply:
Yeah, I do find this a bit bothersome from a consistency standpoint but I also don't have a good alternative...